src/api/api_test.go

```diff
@@ -369,7 +368,7 @@ func (c testClient) url(resp *http.Response) string {
 	return resp.Request.URL.Path
 }
 
-func url(url_items ...interface{}) string {
+func _url(url_items ...interface{}) string {
```

Comment:
Something changed in 1.6 and now functions conflict with imports across the same package. (We're importing `net/url` in a few other places, so we can't use the same name here. :frowning: